### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by VladaProskura**
- [DEMO LINK](https://VladaProskura.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://VladaProskura.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Consider deploying your solution

**Comment by VladaProskura** [comment]
> Consider deploying your solution

Please, explain in more detail what needs to be done?

File: src/style.css [hunk 1]
```diff
@@ -1 +1,111 @@
-/* add styles here */
+
```
Comment (OleziO) [inline src/style.css:1]:
```suggestion
```

File: src/style.css [hunk 2]
```diff
@@ -1 +1,111 @@
...
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.form__input {
+  width: 100%;
+  height: 100%;
+  border: 1px solid #E1E7ED;
```
Comment (OleziO) [inline src/style.css:68]: Move all repeatable colors to the CSS variables.

[More info](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)

**Review by OleziO — CHANGES_REQUESTED** [review]
image
Fix demo link.

File: src/style.css [hunk 3]
```diff
@@ -1 +1,117 @@
 /* add styles here */
```
Comment (OleziO) [inline src/style.css:1]: Remove all comments from the code.

**Review by OleziO — CHANGES_REQUESTED** [review]

**Review by etojeDenys — CHANGES_REQUESTED** [review]
still, your links don't work, try to fix that

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]
1. Placeholders have different font-weight on focus 
image
image

**Pull request closed without merging**